src/runner.js

```diff
@@ -170,7 +189,13 @@ class Runner {
       }
 
       const optionResults = new results.OptionsResult(filteredSelections.map((s) => {
-        return s.text;
+        const formatInstance = new MessageFormat(
```

Comment:
should we be creating a new instance of MessageFormat for each message? Why not create it only once - after ln24 where we defined its options